src/repositories/typeorm-repository.ts

```diff
@@ -44,10 +44,13 @@ export class TypeORMRepository<T extends Entity, ID extends string>
     public dataSource: TypeORMDataSource,
   ) {
     this.id_generator = new UniqueIDGenerator();
+    if (this.typeOrmRepo == undefined){
+      this.init()
+    }
   }
 
   async init() {
-    if (this.typeOrmRepo != null) return;
+    if (this.typeOrmRepo != null && this.typeOrmRepo != undefined) return;
```

Comment:
Maybe typeOrmRepo should instead have been initialized to `null` somewhere else? I recommend checking what is in the other repositories that currently work and modify that instead of this file.